examples/GPU/acoustics_2d_adjoint/Makefile

```diff
@@ -22,26 +22,24 @@ PLOTDIR = _plots                    # Directory for plots
 OVERWRITE ?= True                   # False ==> make a copy of OUTDIR first
 RESTART ?= False                    # Should = clawdata.restart in setrun
 
-FC = pgi
 USE_CUDA = TRUE
-# FC = gfortran
-# USE_CUDA = FALSE
+FC = pgi
```

Comment:
It seems like this would be more consistent with our other usage if you set

    FC ?= pgfortran

here and in other application makefiles.  See my comment on https://github.com/clawpack/clawutil/pull/141